### DISCUSSION TITLE
Thermostat screen disable

### DISCUSSION THREAD

**Discussion by Paulbhyo** [PeteRager/lennoxs30#194] [Ideas]
Hello,. Do you think there is a way to enable or disable the thermostat touch screen?.  I didn't see it as an existing control switch.  My kids keep using the thermostat to check the weather and inadvertently change the temperature.  I can automate changing the setting back the the schedule but I wasn't sure if there was a way to just to disable or lock the primary thermostat.

**Comment by PeteRager** [comment]
There is a screen lock function on the panel itself,  not sure if that does what you need.

Alternatively, if you know what it should be set to, you could create an automation that runs periodically and resets the temperature.
